cmd-graph.c

```diff
@@ -125,6 +127,7 @@ static struct display_field field_self_time= {
 static struct display_field field_addr = {
 	.id      = GRAPH_F_ADDR,
 	.name    = "addr",
+	.alias   = "address",
```

Comment:
Minor nitpick.. could you please swap the name and alias of 'addr' for consistency?

Reply:
Sure. I was thinking about it, but thought about the man pages as well. I think we can just simply write aliases only on the man pages for simplicity.

Reply:
ok